third_party/nvfuser/csrc/python_frontend/fusion_record.h

```diff
@@ -1699,7 +1656,20 @@ struct VarianceOpRecord : NormOpRecord {
 
   virtual void operator()(FusionDefinition& fd) final {
     auto arg = fd.getFusionState(args_.at(0).index)->as<Nvf::TensorView>();
-    auto output = Nvf::variance(arg, axes_, correction_, keep_dim_);
+    auto vec = fd.getFusionStateVector(args_.at(1).index);
```

Comment:
comment to myself, this conversion from `FusionStateVector` to `std::vector<int>` should be put inside a util function under `FusionDefinition`.